internal/monitors/expvar/expvar.go

```diff
@@ -18,6 +19,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const monitorType = "expvar"
```

Comment:
This is perhaps leftover code in your clone from when you did #696? Should be removed here.